src/pkg/caendr/caendr/models/datastore/file_record_entity.py

```diff
@@ -24,6 +24,7 @@ def get_props_set(cls):
     return {
       *super().get_props_set(),
       'filename',
+      'hashed_filename',
```

Comment:
when naming things, in this case, prefer most-important-to-least-important; ie. `filename_hash`

Reply:
Pushed in the commit cfcc1cb3f94f015cf32a978d11ee26caa2409f32